packages/ui/components/checkbox-group/src/LionCheckboxIndeterminate.js

```diff
@@ -30,6 +30,9 @@ export class LionCheckboxIndeterminate extends LionCheckbox {
         type: Boolean,
         reflect: true,
       },
+      forceSyncIndeterminate: {
```

Comment:
Could you update tests and docs for this public API?

Reply:
Should it be public?